lib/emerald/nodes/list.rb

```diff
@@ -4,22 +4,15 @@
 require 'treetop'
 require_relative 'node'
 
-# TODO: Refactor this and update doc comment
-# Multiple head elements I think
-class List < Node
+# Special rule for lists of images, styles, and scripts
+class ListSpecial < Node
```

Comment:
Hey, I already have an `attr_list` rule in the grammar. What do you think I should call it instead?